applications/nrf_desktop/configuration/nrf52810_pca20045/app_ZDebug.conf

```diff
@@ -19,7 +19,7 @@ CONFIG_DESKTOP_WHEEL_ENABLE=y
 CONFIG_DESKTOP_WHEEL_SENSOR_VALUE_DIVIDER=15
 
 CONFIG_DESKTOP_MOTION_PAW3212_ENABLE=y
-CONFIG_DESKTOP_MOTION_THREAD_STACK_SIZE=1024
+CONFIG_DESKTOP_MOTION_THREAD_STACK_SIZE=512
```

Comment:
Is this safe? I see it is still 1024 on the pca20041. Just making sure that it is done on purpose.

Reply:
Yes, this was done on purpose. I think only problem with lower stack size may be observed for pca20041 on log panic (logs will be processed from context of the call - e.g. motion thread). For pca20045 logger is disabled, so this problem should not occur.